plugins.js

```diff
@@ -0,0 +1,51 @@
+module.exports = [
+  {
+    name: 'postcss-simple-vars',
+  },
+  {
+    name: 'postcss-advanced-variables',
+    config: { disable: ['@import'] },
+    url: 'https://github.com/jonathantneal/postcss-advanced-variables',
+  },
+  {
+    name: 'postcss-mixins',
+  },
+  {
+    name: 'postcss-nesting',
+  },
+  {
+    name: 'postcss-nested',
+  },
+  {
+    name: 'short',
+    pkgName: 'postcss-short',
+  },
+  {
+    name: 'rucksack',
+    pkgName: 'rucksack-css',
+  },
+  {
+    name: 'lost',
+  },
+  {
+    name: 'cssnext',
+    pkgName: 'postcss-cssnext',
```

Comment:
The name here should be `postcss-cssnext`. `cssnext` refers to an older project by the same author that did the same thing, but was _not_ a postcss plugin. Ref: http://cssnext.io/postcss/

Reply:
same than previous comment. What should we do ? preferredName ?

Reply:
Man, IDK, this is a mess.

Reply:
The npm users have spoken. cssnext has 14,738 in the last month. postcss-cssnext has 617,906. There is no github project for cssnext, as it was renamed to postcss-cssnext.

Long live postcss-cssnext.

Reply:
Refs https://github.com/himynameisdave/postcss-plugins/issues/229